sycl/include/sycl/ext/intel/experimental/fpga_annotated_properties.hpp

```diff
@@ -1,4 +1,5 @@
-//==-- properties.hpp - SYCL properties associated with annotated_arg/ptr --==//
+//==-- fpga_annotated_properties.hpp - SYCL properties associated with
+// annotated_arg/ptr --==//
```

Comment:
@broxigarchen, formatting if off here and in sycl/include/sycl/ext/oneapi/experimental/common_annotated_properties/properties.hpp.